modules/animation/include/modules/animation/workspaceanimations.h

```diff
@@ -96,8 +97,10 @@ class IVW_MODULE_ANIMATION_API WorkspaceAnimations : public AnimationControllerO
     Animation& operator[](size_t i);
     const Animation& operator[](size_t i) const;
 
-    std::vector<Animation>::const_iterator begin() const;
-    std::vector<Animation>::const_iterator end() const;
+
+    using iterator = util::IndirectIterator<std::vector<std::unique_ptr<Animation>>::const_iterator>;
```

Comment:
```suggestion
    using const_iterator = util::IndirectIterator<std::vector<std::unique_ptr<Animation>>::const_iterator>;
```

Reply:
unless we add a non-const version I don't see the need.